components/ledger/internal/engine/resolver.go

```diff
@@ -67,6 +69,15 @@ func NewResolver(storageDriver *driver.Driver, options ...option) *Resolver {
 	return r
 }
 
+func (r *Resolver) startLedgerUsingStore(ctx context.Context, name string, store *ledgerstore.Store) *Ledger {
```

Comment:
start is called in the getLedger and also in the create ? Should we have a way to know if the ledger is already started or not in order to not launch the same goroutine ?

Reply:
My bad, it's not launch in the get since you search in the map before, discard this comment